src/Microsoft.IdentityModel.Tokens/ECDsaAdapter.cs

```diff
@@ -52,12 +52,14 @@ internal class ECDsaAdapter
         /// </exception>
         internal ECDsaAdapter()
         {
-#if NETSTANDARD2_0
+#if NET472
+            CreateECDsaFunction = CreateECDsaUsingECParams;
+#elif NETSTANDARD2_0
```

Comment:
@mafurman Could you add a comment here that although NETSTANDARD2_0 specifies that ECParameters is supported, we still to call SupportsECParameters() as 4..6.1 is listed as supporting NETSTANDARD2_0, but DOES NOT support ECParameters. see: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecparameters?view=netstandard-2.0

This may help future devs as they may not understand the issues.